ext/net/02_tls.js

```diff
@@ -61,20 +61,35 @@ async function connectTls({
       new Error().stack,
       "Pass the cert file contents to the `Deno.ConnectTlsOptions.cert` option instead.",
     );
+    if (internals.future) {
+      throw new TypeError(
+        "`Deno.ConnectTlsOptions.certFile` has been removed in favor of `Deno.ConnectTlsOptions.cert`.",
+      );
+    }
   }
   if (certChain !== undefined) {
     internals.warnOnDeprecatedApi(
       "Deno.ConnectTlsOptions.certChain",
       new Error().stack,
       "Use the `Deno.ConnectTlsOptions.cert` option instead.",
     );
+    if (internals.future) {
+      throw new TypeError(
+        "`Deno.ConnectTlsOptions.certChain` has been removed in favor of `Deno.ConnectTlsOptions.cert`.",
+      );
+    }
```

Comment:
Maybe instead of throwing an error we just make it undefined so it doesn't work (as would happen in Deno 2)